src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateNotificationService.swift

```diff
@@ -180,14 +180,15 @@ class HealthCertificateNotificationService {
 		completion: @escaping () -> Void
 	) {
 		let name = person.name?.standardizedName
-		guard let newCertificateReissuance = person.dccWalletInfo?.certificateReissuance else {
+		guard let newCertificateReissuance = person.dccWalletInfo?.certificateReissuance,
+			  let newIdentifier = newCertificateReissuance.reissuanceDivision.identifier else {
 			Log.info("No certificate reissuance found for person \(private: String(describing: name))", log: .vaccination)
 			completion()
 
 			return
 		}
-
-		if newCertificateReissuance != previousCertificateReissuance {
+		
+		 if newIdentifier != previousCertificateReissuance?.reissuanceDivision.identifier ?? "renew" {
```

Comment:
Just to document our discussion: The fallback should only be for the case that the `identifier` is nil, not if the `previousCertificateReissuance` itself is nil

Reply:
@nickguendling  updated 👍